src/main/resources/db/migration/changelog-202304281059.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="loretdemolas" id="202304281059-01">
+        <createTable tableName="Notifications">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="type_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="body" type="VARCHAR(1000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="isRead" type="BOOLEAN" defaultValueBoolean="false">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_date" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false" />
+            </column>
+            <column name="last_updated_date" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="loretdemolas" id="202304281059-02">
+        <createTable tableName="Notification_Type">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="icon_url" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="loretdemolas" id="202304281059-03">
+        <addForeignKeyConstraint
+                baseTableName="Notifications"
+                baseColumnNames="type_id"
+                referencedTableName="Notification_Type"
+                referencedColumnNames="id"
+                constraintName="notifications_type_id_fk"
+        />
+    </changeSet>
+
+    <changeSet author="loretdemolas" id="202304281059-04" context="test">
```

Comment:
Use two changesets.. one for notification_type, and the other for test notification data. Also, I think we only need two notification types now, one "Attribute request approved", and one "Attribute request rejected". We may change that wording eventually.

And the notification types should be in a changeSet without `context=true`; they should always be there.

Reply:
Fixed, I had context="test" is this correct or should I change it to context=true;